tensorflow/python/ops/image_ops_impl.py

```diff
@@ -626,6 +650,18 @@ def _rot270():
 @tf_export('image.transpose', v1=['image.transpose', 'image.transpose_image'])
 def transpose(image, name=None):
   """Transpose image(s) by swapping the height and width dimension.
+    
+  Usage Example:
+    >>> x = [[[1.0, 2.0, 3.0],
```

Comment:
One last change. 

Please add a new line before and after the doctest. This won't affect the results of the run, but it won't render properly on the website. Also, indent it at the same level as "Usage example:"

Something like this:

```
Usage example:

>>> doctest
output

More text
```

Reply:
Ok I will do, please let me at least see the successfull build on Ubuntu CPU, I've been waiting too long for it and if I do the change now, it will stop building :D

Reply:
Sounds good to me.

Reply:
Thanks!

Reply:
Oh my, it really gave a successful build! Now time to do the last change.

Reply:
Can you also indent it to the same level as `Usage Example`?

Reply:
Oh yeah, sure, forgot about that one.

Reply:
Sorry about forgetting that one, done now -> https://github.com/tensorflow/tensorflow/pull/35388/commits/5f6f507d971bca97152d9ebccc7b2f53772cb011